backend/open_webui/routers/users.py

```
@@ -309,6 +330,20 @@ async def update_user_by_id(
+            credit = Credits.set_credit_by_user_id(
+                SetCreditForm(
+                    user_id=user_id,
+                    credit=Decimal(form_data.credit),
+                    detail=SetCreditFormDetail(
+                        api_path=str(request.url),
+                        api_params={"credit": form_data.credit},
+                        desc=f"updated by {session_user.name}",
+                    ),
+                )
+            )
+            setattr(updated_user, "credit", "%.4f" % credit.credit)
```

Ruff (0.8.2)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)